akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -227,6 +228,7 @@ object Attributes {
 
   val none: Attributes = Attributes()
 
+  val verboseLogging: Attributes = Attributes(VerboseLogging)
```

Comment:
Think this description should work....part of what I was adding for was getting buffer information, so wanted to include that note as well.